src/ServiceControlInstaller.Engine/Instances/BaseService.cs

```diff
@@ -74,7 +74,7 @@ public bool TryStartService()
 
             try
             {
-                Service.WaitForStatus(ServiceControllerStatus.Running, TimeSpan.FromSeconds(30));
+                Service.WaitForStatus(ServiceControllerStatus.Running, TimeSpan.FromSeconds(120));
```

Comment:
Is this change necessary for the new transports?

Reply:
Not necessary, but there were times where it took longer than 30 seconds on my machine to start the service. Happy to revert though?